gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -107,6 +212,21 @@ def main(args):
         )
         logger.info("CADD Hail Table for GRCh38 created.")
 
+    if args.revel:
+        logger.info("Creating REVEL Hail Table for GRCh38...")
+        # TODO: update path, do we keep both the raw and processed files?
```

Comment:
I think if we document where we got these from and if they remain available, we do not need to. If they aren't too large, we could just store them in a more general bucket..

Reply:
REVEL one is not only ~525M, but it's still downloadable.